utmpdump: add page (#2931)
tldr-pages/tldr: pages/linux/utmpdump.md

# utmpdump

> Dump and load btmp, utmp and wtmp accounting files.

> - Dump {{/var/log/wtmp}} file to standard out as plain text:

Reviewer: Perhaps this will read better?
```suggestion
- Dump the `/var/log/wtmp` file to the standard output as plain text:
```

`utmpdump {{/var/log/wtmp}}`

> - Load previously dumped {{dumpfile}} into {{/var/log/wtmp}:

Reviewer: Perhaps this will read better?
```suggestion
- Load a previously dumped file into `/var/log/wtmp`:
```

> `utmpdump -r {{dumpfile}} > {{/var/log/wtmp}`

Reviewer: Oops! I missed this one. Sorry about that!
```suggestion
`utmpdump -r {{dumpfile}} > {{/var/log/wtmp}}`
```
